components/attention/w-attention.vue

```diff
@@ -27,92 +24,139 @@ const props = defineProps({
   ...modelProps({ modelDefault: absentProp }),
   targetEl: Object,
   attentionClass: [Object, String],
-  attentionEl: {
-    default: () => ref(null),
-  },
   role: String,
   ariaLabel: String,
+  placement: {
+    type: String,
+    validator(value) {
+      return [
+        'top-start',
+        'top',
+        'top-end',
+        'right-start',
+        'right',
+        'right-end',
+        'bottom-start',
+        'bottom',
+        'bottom-end',
+        'left-start',
+        'left',
+        'left-end'].includes(value)
+    },
+    default: 'bottom'
+  },
+  distance: {
+    type: Number,
+    default: 8
+  },
+  skidding: {
+    type: Number,
+    default: 0
+  },
+  flip: {
+    type: Boolean,
+    default: false
+  },
+  fallbackPlacements: {
+    type: Array,
+    default: undefined,
+    validator(values) {
+      const acceptedValues = [
+        'top-start',
```

Comment:
Same list as above. Maybe worth extracting it above as a const and using it here?

Reply:
I removed it and instead used `directions` from the `core` library. :) Updated in this commit: [bda07aa](https://github.com/warp-ds/vue/pull/141/commits/bda07aa92ade74b7131a4992b592a0e6a70efa98)